install/tools/ipa-ca-install

```diff
@@ -71,6 +78,16 @@ def parse_options():
                       help="Signing algorithm of the IPA CA certificate")
     parser.add_option("-P", "--principal", dest="principal", sensitive=True,
                       default=None, help="User allowed to manage replicas")
+    parser.add_option("--subject-base", dest="subject_base",
+                      default=None,
+                      help=(
+                          "The certificate subject base "
+                          "(default O=<realm-name>)"))
```

Comment:
IMO this deservers a separate "add missing --subject-base to ipa-ca-install", because the option should have been there since `ipa-ca-install` gained the ability to install CA master.